[UK - UKTVPLAY] Fix for replay

nictjir opened the pull request:
Dynamically find file name containing playerID

nictjir committed:
[UK - UKTVPLAY] Fix for replay 

Dynamically find file name containing playerID

joaopa00 commented:
You did not update your private repository after I merged and modified your first commit. So changes were not taken in account. Please push-force a new commit after updatign your private repo.

nictjir commented:
Trying again from scratch

nictjir commented:
Deleted my fork,
readded a fork
modified
 blaze.py
 uktvplay.py
in my repo 
and did a pull request

and it fails again
Run python main.py --test-modules -a ../plugin.video.catchuptvandmore
Error: The operation was canceled.

Not sure what i have done incorrectly
